app/apphandlers/embeddedAppHandler.go

```diff
@@ -219,7 +232,7 @@ func (a *embeddedApp) StartRudderCore(ctx context.Context, options *app.Options)
 		Reporting:        reportingI,
 		BackendConfig:    backendconfig.DefaultBackendConfig,
 		RouterDB:         routerDB,
-		ProcErrorDB:      errDB,
+		ProcErrorDB:      errDBForRead,
```

Comment:
Except for processor, which both reads (stash loop) and writes, all other components are only writing to `proc_errors`, correct?

Reply:
Oops. Yes, this must be errDBForWrite.

Reply:
Oops. Yes you are right.

Both rt & brt should recive errDBForWrite and not Read. Corrected.